development/common/logs_utils.py

```diff
@@ -0,0 +1,28 @@
+
+from development.constants import (
+    ACTION,
+    ALL_ACTIONS,
+    ALL_STATES,
+    STATE,
+)
+
+
+class FilterLogs:
+
+    def __init__(self, data: list) -> None:
+        self.logs = data
+
+    @property
+    def possible_states(self) -> list:
+        possible_states = []
+        for log in self.logs:
+            if STATE in log.keys():
+                possible_states.append(log[STATE])
+        possible_states.append(ALL_STATES)
```

Comment:
There is a pattern (comprehension list [more info](https://www.w3schools.com/python/python_lists_comprehension.asp)) in Python to replace the append function which is this one:
```suggestion
        possible_states = [
            log[STATE]
            for log in self.logs
            if STATE in log.keys()
        ]
```

Reply:
Or in one-line is
```
possible_actions = [log[STATE] for log in self.logs if STATE in log.keys()]
```
If you see the method `possible_actions` already has it implemented

Reply:
Yes, that was my mistake. In the previous  [PR](https://github.com/evbeda/edagames-django/pull/188) I add the possible_states and put it in the wrong way I pretend to refactor it in a new PR  instead of this
But the previous PR it's still open so I refactor it there. 
Thank you